doc/developers/develop.rst

```diff
@@ -496,6 +496,14 @@ no_validation (default=False)
 non_deterministic (default=False)
     whether the estimator is not deterministic given a fixed ``random_state``
 
+preserves_dtype (default=``[np.float64]``)
+    dtypes for `X` that an estimator will preserve. Possible options are a
```

Comment:
I think we should clarify what "preserving dtypes" means, and also indicate that this is only relevant for transformers.